mbed_greentea/mbed_greentea_cli.py

```diff
@@ -281,6 +281,11 @@ def main():
                     default=None,
                     help='Shuffle seed (If you want to reproduce your shuffle order please use seed provided in test summary)')
 
+    parser.add_option('', '--sync',
+                    dest='sync_packet',
+                    default=5,
```

Comment:
Would you like to increase this value to 5? Or should we keep it at 2 for now? If you'd like to change it to 5, please update the help text below.